### PR TITLE
Db

### PR DESCRIPTION
**Pull request by cthao309**
Hey guys, when you have a chance can someone review my database query for retrieving reviews by restaurant id. Thanks

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -1,6 +1,11 @@
 const express = require('express');
 const path = require('path');
-const db = require('../db/db.js');
```
Comment (mwuf15) [inline server/app.js:3]: you have two variables called db.

File: server/app.js [hunk 2]
```diff
@@ -1,6 +1,11 @@
...
+const db = require('../db/mysql.js');
+
```
Comment (mwuf15) [inline server/app.js]: you have two variables called db.

Comment (cthao309) [reply server/app.js]: not sure why github doesn't have my latest change?

**Comment by cthao309** [comment]
close for testing

**Pull request merged**